src/plugins/dam/components/search-assets-screen/index.js

```diff
@@ -15,6 +16,7 @@ import SearchForm from '@triniti/cms/plugins/dam/components/search-assets-screen
 import BatchOperationsCard from '@triniti/cms/plugins/dam/components/search-assets-screen/BatchOperationsCard.js';
 import useBatch from '@triniti/cms/plugins/ncr/components/useBatch.js';
 import AssetIcon from '@triniti/cms/plugins/dam/components/asset-icon/index.js';
+import nodeClickNavigator from '@triniti/cms/utils/nodeClickNavigator.js';
```

Comment:
@rsumilang can you call it `createRowClickHandler.js` and the data attribute to `data-ignore-row-click`